packages/synapse-react-client/src/components/SynapseTable/SynapseTableRenderers.tsx

```diff
@@ -142,8 +142,7 @@ const getEntityOrRowId = (
   props: CellContext<Row, unknown>,
 ): string | undefined => {
   const { row, table } = props
-  const rowEntityIDColumnIndex: number | undefined = (table.options.meta as any)
-    .rowEntityIDColumnIndex
+  const rowEntityIDColumnIndex = table.options.meta?.rowEntityIDColumnIndex
```

Comment:
added these meta variables to tanstack-table-augmentation.d.ts so we can use them without casting